mne/viz/topomap.py

```diff
@@ -978,7 +987,7 @@ def _plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
 
     # find mask limits
     extent, Xi, Yi, interp = _setup_interp(
-        pos, res, extrapolate, sphere, outlines, border)
+        pos, res, image_interp, extrapolate, sphere, outlines, border)
```

Comment:
the lines below (`set_values` and `set_locations`) should be done only for linear and cubic interpolation. Voronoi does not use the interpolated grid `Zi`.

Reply:
You have to get the extent though...

Reply:
`Zi` is used for the contours too...

Reply:
right, the contours... but in such case iI don't think it makes much sense to plot cubic-interpolated contours, and nearest interpolated topomap. One option is to turn off the contours for nearest interpolation - at least for now.